.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  goreleaser:
```

Comment:
I could not find an alternative to this tool. The current version is 1.142.0, meaning that it could break compatibility at any moment.